src/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { rules } from "../rule";
+
+rules.push({
```

Comment:
are we not extending "Rule" to create "DescriptionMustNotBeNodeName" as an implemenation of "Rule"?